Description: Add cpu feature sgx-exinfo.
Author: Tim Wiederhake <twiederh@redhat.com>
Origin: upstream, https://gitlab.com/libvirt/libvirt/-/commit/4b5c9678d17f135e77ddab7c2643949fb2408048
Bug-Ubuntu: https://launchpad.net/bugs/2028057
Forwarded: not-needed
Last-Update: 2023-11-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/cpu_map/x86_features.xml
+++ b/src/cpu_map/x86_features.xml
@@ -418,6 +418,11 @@
     <cpuid eax_in='0x0f' ecx_in='0x01' edx='0x00000004'/>
   </feature>
 
+  <!-- cpuid level 0x00000012 ecx 0x0 features -->
+  <feature name='sgx-exinfo'>
+    <cpuid eax_in='0x12' ecx_in='0x00' ebx='0x00000001'/>
+  </feature>
+
   <!-- extended features, EDX -->
   <feature name='syscall'>
     <cpuid eax_in='0x80000001' edx='0x00000800'/>
